plan/join_reorder.go

```diff
@@ -47,7 +47,7 @@ func findColumnIndexByGroup(groups []LogicalPlan, col *expression.Column) int {
 			return i
 		}
 	}
-	log.Errorf("Unknown columns %s, from id %d, position %d", col, col.FromID, col.Position)
+	log.Errorf("Unknown columns %s, from id %v, position %d", col, col.FromID, col.Position)
```

Comment:
why change "%d" to "%v" ?

Reply:
my local go vet will report "plan/join_reorder.go:50: arg col.FromID for printf verb %d of wrong type: string"